### ISSUE TITLE
Add `aria-describedby` attribute/functionality to va-button

### ISSUE THREAD

**Issue by powellkerry** [department-of-veterans-affairs/vets-design-system-documentation#2739]
## Description
During a PR review a team chose to use a standard HTML button because there was no support for the `aria-describedby` attribute. This attribute was required by accessibility for their use case. We need to add the ability to set this attribute.

PR reference: https://github.com/department-of-veterans-affairs/vets-website/pull/29188

## Acceptance Criteria
- [ ] Accessibility signs off
- [ ] Screen reader announces correctly

**Comment by rsmithadhoc** [comment]
I think we can add this since there are some situations where it is valid to use this. However, I think we should document some guidance around its usage since this isn't something that should be used all the time (descriptive button text is preferred), so I will create an additional ticket to update the guidance.

**Comment by artsymartha68** [comment]
Hi design system team, can you provide an estimate when this might be addressed? Reason I'm asking is the health portal team is planning their work and would love to have a fix earlier if possible (I know you never hear this from anyone!). If they need to wait then they need to estimate their workload differently. Thanks for your consideration.

**Comment by rsmithadhoc** [comment]
> Hi design system team, can you provide an estimate when this might be addressed? Reason I'm asking is the health portal team is planning their work and would love to have a fix earlier if possible (I know you never hear this from anyone!). If they need to wait then they need to estimate their workload differently. Thanks for your consideration.

@artsymartha68 We will work on this in our next sprint that is starting tomorrow.

**Comment by artsymartha68** [comment]
Y'all are awesome, thanks so much!

**Comment by rsmithadhoc** [comment]
Hi @artsymartha68,

This is available on production. We added a `message-aria-describedby` property to the component where a message can be passed in to read out when that field is tabbed to (but the message is hidden visually). It doesn't function exactly like `aria-describedby` due to shadow DOM restrictions, but the end result should be the same.

Thanks!
